ush/forecast_postdet.sh

```diff
@@ -192,7 +192,14 @@ EOF
   fi
 
 # NoahMP table
-  ${NLN} "${HOMEgfs}/parm/ufs/land/noahmptable.tbl" "${DATA}/noahmptable.tbl" 
+      local noahmptablefile="${HOMEgfs}/sorc/ufs_model.fd/FV3/ccpp/physics/physics/noahmptable.tbl"
```

Comment:
See above comment. The location before was correct. The additional error checking is good though.